brambling/tests/functional/test_order_forms.py

```diff
@@ -219,6 +218,20 @@ def test_dwolla_payment_form(self, dwolla_charge, dwolla_get_sources):
         self.assertEqual(Decimal(str(txn.application_fee)), Decimal('1.05'))
         self.assertEqual(Decimal(str(txn.processing_fee)), Decimal('0.25'))
 
+    @patch('brambling.forms.orders.dwolla_get_sources')
+    @patch('brambling.forms.orders.dwolla_charge', side_effect=ValueError('this is an error'))
```

Comment:
This seems slightly artificial to me. I'd prefer something that mimics the error this code change is meant to handle more exactly, e.g. (omitting unchanged lines)

```python
@patch('dwolla.oauth.refresh')
def test_dwolla_payment_form_handles_valueerror(self, refresh, dwolla_get_sources):
    refresh.return_value = {u'error': u'access_denied', u'error_description': u'Invalid refresh token.'}
```